tests/cases/fourslash/findAllRefs_importType_js.ts

```diff
@@ -18,6 +18,11 @@ verify.noErrors();
 // TODO: GH#24025
 
 const [rModuleDef, rModule, r0Def, r0, r1Def, r1, r2Def, r2, r3Def, r3, r4Def, r4, r5] = test.ranges();
+verify.referenceGroups([r3, r4], [

```

Comment:
I don't understand why the fourslash tests are changing.

Reply:
our fourslash ls host doesnt use getProjectVersion, Which means it was having the issue we are fixing here.. THat is it was updating program too many times (since version for only root files is correctly returned to check and other files are always not correctly versioned) Which meant that every findAllReferences used new program and the types were not cached.. This is bug in findAllReferences with `module.exports` where depending on expandoes calculation the results differ.. Dont want to fix that issue as part of this and will file an issue for that..

Reply:
Created #36988